src/EFCore.MySql/Storage/Internal/MySqlOldGuidTypeMapping.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Pomelo Foundation. All rights reserved.
+// Licensed under the MIT. See LICENSE in the project root for license information.
+
+using System;
+using JetBrains.Annotations;
+using Pomelo.EntityFrameworkCore.MySql.Utilities;
+
+namespace Microsoft.EntityFrameworkCore.Storage
+{
+    /// <summary>
+    ///     This API supports the Entity Framework Core infrastructure and is not intended to be used
+    ///     directly from your code. This API may change or be removed in future releases.
+    /// </summary>
+    internal sealed class MySqlOldGuidTypeMapping : GuidTypeMapping
+    {
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="MySqlOldGuidTypeMapping" /> class.
+        /// </summary>
+        public MySqlOldGuidTypeMapping() : base("binary(16)", System.Data.DbType.Guid) { }
```

Comment:
You should call the constructor that takes a `RelationalTypeMappingParameters` and override the `Clone` methods. This will make this mapping compatible with new EF features.

Reply:
Ok updated. I have followed the pattern present in the other classes.